classes/profile_table.php

```diff
@@ -280,7 +280,7 @@ public function col_user(object $record): string {
     public function col_responsecode(object $record): string {
         if ($this->is_downloading()) {
             return $record->responsecode;
-        } else if ($record->scripttype == profile::SCRIPTTYPE_CLI) {
+        } else if ($record->responsecode <= 100) {
```

Comment:
not quite here, but for cli can you remove the redundant -

![image](https://user-images.githubusercontent.com/187449/148167420-9fdabc58-7e0c-4f9f-8316-444f32038667.png)

Reply:
fixed